railties/test/generators/actions_test.rb

```diff
@@ -141,7 +141,7 @@ def test_gem_group_should_wrap_gems_in_a_group
       gem "fakeweb"
     end
 
-    assert_file "Gemfile", /\ngroup :development, :test do\n  gem 'rspec-rails'\nend\n\ngroup :test do\n  gem 'fakeweb'\nend/
+    assert_file "Gemfile", /^group :development, :test do\n  gem 'rspec-rails'\nend\ngroup :test do\n  gem 'fakeweb'\nend\n\z/
```

Comment:
Should not add newline between `group`s?

Reply:
@kamipo 

I understand that we should add a newline between `group`s, `github`s and `add_source`s.
I will create an another PR.

Reply:
@kamipo 
I understand that it should.
Thank you for pointing it out!

Here's the PR for it🙏
https://github.com/rails/rails/pull/36723